README.md

```diff
@@ -32,8 +32,8 @@ They may be separated from the repository at some point (and possibly released o
 ## Build & Run
 
 1. `cargo build`
-2. Put a [`default.toml`](application/comit_node/config/default.toml) config file into `~/.config/comit_node` or pass `--config <config_path>` with `<config_path>` as the folder path to where the `default.toml` is located.
-3. Put a [`default.toml`](application/btsieve/config/default.toml) config file into `~/.config/btsieve` or set `BTSIEVE_CONFIG_PATH` as folder path to where the `default.toml` is located
+2. Put a [(default)](comit_node/config/comit_node.toml) config file into `~/.config/comit/comit_node.toml` or pass `--config <config_file>`.
```

Comment:
Well actually this file is generated automatically so I think we could just skip this step or mention it's optional.

Reply:
Good point, will update PR.

Reply:
Done, thanks.